apps/prairielearn/src/pages/instructorStudentDetail/instructorStudentDetail.html.tsx

```diff
@@ -72,210 +67,41 @@ export function InstructorStudentDetail({
   };
 
   return (
-    <>
-      <div class="card mb-4">
-        <div class="card-header bg-primary text-white d-flex align-items-center justify-content-between">
-          <h1 class="mb-0">Details</h1>
-          {user && enrollment.status === 'joined' && (
-            <button type="button" class="btn btn-sm btn-light" onClick={handleViewAsStudent}>
-              <i class="fas fa-user-graduate me-1" aria-hidden="true" />
-              View as student
-            </button>
-          )}
-        </div>
-        <div class="card-body">
-          <div class="d-flex align-items-center justify-content-between">
-            <h2 class="d-flex align-items-center">
-              {user?.name ?? enrollment.pending_uid}
-              {enrollment.status !== 'joined' && (
-                <EnrollmentStatusIcon type="badge" class="ms-2 fs-6" status={enrollment.status} />
-              )}
-            </h2>
-            {hasCourseInstancePermissionEdit && enrollmentManagementEnabled && (
-              <div class="d-flex gap-2 align-items-center">
-                {enrollment.status === 'joined' && (
-                  <form method="POST">
-                    <input type="hidden" name="__csrf_token" value={csrfToken} />
-                    <input type="hidden" name="__action" value="block_student" />
-                    <button type="submit" class="btn btn-sm btn-outline-danger">
-                      <i class="fas fa-user-slash me-1" aria-hidden="true" /> Block student
```

Comment:
Because I can't comment on what looks to be the actual implementation: putting this all on one line means you end up with double space between the icon and the text: one literal space, and the `me-1` spacing.

Two options:

- Use `d-flex flex-row align-items-center gap-[something]`
- Move "Block students" to a new line so that Preact doesn't render the extra space

Ditto for other buttons below.